opam

```diff
@@ -41,7 +41,7 @@ depends: [
     "bitstring"
     "camlzip"
     "cmdliner" {>= "0.9.6"}
-    "core_kernel" {>= "112.17.0" & < "112.35.0"}
+    "core_kernel" {>= "112.35.0" & < "113.00.0"}
```

Comment:
can you, please, split this in two different PRs?
One that adds `Bytes` module, and the other, that actually bumps the version up.